www/build-themes.js

```diff
@@ -13,14 +13,7 @@ const importer = function importer(url, prev) {
     // since stylesheets there also use tilde imports which
     // webpack resolves to Paragon's node_modules (not node_modules related to docs site),
     // so we do the same
```

Comment:
I think this comment is also not needed anymore, since it talks about `www`'s `node_modules` (which we do not have anymore), we can delete it.

Reply:
Done